apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx

```diff
@@ -38,9 +38,13 @@ export default class LandingPage extends Component {
           )}
           <EnrolledWorkshops />
           {this.props.deeperLearningCourseData && (
-            <ProfessionalLearningCourseProgress
-              deeperLearningCourseData={this.props.deeperLearningCourseData}
-            />
+            <div>
+              <Heading2>Online Professional Learning Courses</Heading2>
```

Comment:
I didn't translate this text because it's temporary and will be removed when we add the facilitator page.